### ISSUE TITLE
hi, can u achieve the paper results?

### ISSUE THREAD

**Issue by oujieww** [IgorMunizS/MultiPoseID#1]
MultiPoseNet, i have many questions about that paper, especially for the inference time of that paper.
how do u think about those questions ? can u help me ?

in paper's abstract ,"the fastest real time system with ∼23 frames/sec". And we can find "Keypoint and person detections take 35 ms while PRN takes 2 ms per instance" in 4.5 runtime analysis ,So we can get a result 1000/(40) ~=23 fps.

1."is the 35ms include "load image, resize image, transform image to tensor, normlized the image , put the image to cuda, and model inference" ? or "35 ms only for model inference?"

2.i test the speed of PRN , is same as reported on paper 2ms/instance ,but how about the time for "select the box from the retinanet" , "crop the feature map for every instance " and "resize the every instance " and the time for "cuda heatmaps for instance transformed to keypoints coordinate"

"load image , resize , to tensor ....." "select box , nms, crop heatmaps for everyone " cost too many time..., So ,how long your code take for those operation?

**Comment by oujieww** [comment]
^_^ thank u !!!

**Comment by IgorMunizS** [comment]
Hi oujieww,

Actually I didn't make inference time tests yet, so i'm not able to discuss the paper results.
But it is important to note that the results were obtained in a GTX1080Ti GPU, the same GPU or equivalent one is necessary for us to have a notion.

1."is the 35ms include "load image, resize image, transform image to tensor, normlized the image , put the image to cuda, and model inference" ? or "35 ms only for model inference?"

In my understanding, the authors mention that the complete system has ~ 23 average FPS. So in that value I would include all procedures. 

For now I'm working on improving accuracy results. There is only one implementation in pytorch that obtained similar results, but slightly worse.
When I finish, I'll be glad to do the inference time tests. 
If you are trying to replicate the paper in Keras too, we can share some ideas.

**Comment by oujieww** [comment]
@IgorMunizS  thanks for your respond. i use GTX1080Ti.   but i use the pytorch , based on LiMeng's  repo...
